components/Footer.tsx

```diff
@@ -0,0 +1,30 @@
+import '@fontsource/roboto';
+import { Grid, Typography } from '@mui/material';
+
+export function Footer() {
+  return (
+    <footer>
+      <Grid
+        container
+        direction='column'
+        alignItems='center'
+        justifyContent='center'
+        bgcolor='#1976D2'
+        spacing={0}
+        padding={0}
+        height={158}
+      >
+        <Grid
+          style={{ textAlign: 'center' }}
+          color='#ffffff'
+        >
+          <div style={{ alignItems: 'center', display: 'flex', gap: 4 }}>
+            <Typography sx={{ fontWeight: '500' }}>Copyright © 2022 </Typography>
```

Comment:
for **fontWeight**, please use function and appropriate value from default theme that we are using:
[default theme](https://mui.com/material-ui/customization/default-theme/)

Example of how to use default theme
```tsx
<Button sx={{ marginBottom: theme => theme.spacing(3)}} />
```